src/index.js

```diff
@@ -278,9 +280,11 @@ class RuntimeMetrics {
   }
 
   _cpuHeap() {
-    RuntimeMetrics.measureCpuHeap(this);
-    const reg = this.registry;
-    this._intervals.push(reg.schedulePeriodically(RuntimeMetrics.measureCpuHeap, 60000, this));
+    setTimeout(() => {
+      RuntimeMetrics.measureCpuHeap(this);
+      const reg = this.registry;
+      this._intervals.push(reg.schedulePeriodically(RuntimeMetrics.measureCpuHeap, 60000, this));
+    }, this.cpuGaugeStartDelay);
```

Comment:
Do we want to unref this timer so that it wouldn't potentially block a process exit?